dev/buildtool/cloudbuild/containers.yml

```diff
@@ -49,3 +52,4 @@ options:
 substitutions:
   _COMPILE_CACHE_BUCKET: spinnaker-build-cache
   _BRANCH_TAG: unknown
+  _ARTIFACT_REGISTRY: us-docker.pkg.dev/spinnaker-community/nightly
```

Comment:
Probably this should be nightly-containers or nightly-docker or something?

Reply:
Sorry, I missed this comment before submitting. Is the `us-docker` prefix not enough to suggest it's for Docker containers?